packages/marshal/test/test-marshal-capdata.js

```diff
@@ -6,6 +6,11 @@ import { makeMarshal } from '../src/marshal.js';
 
 const { freeze, isFrozen, create, prototype: objectPrototype } = Object;
 
+const harden = /** @type {import('ses').Harden & {isFake: boolean}} */ (
```

Comment:
Would the following also work? If so, I think it better documents the runtime reality it describes. Likewise for the other occurrences of course

```suggestion
const harden = /** @type {import('ses').Harden & {isFake?: boolean}} */ (
```

Reply:
Yes it would. I'll see about adding tacking that onto another PR.

Reply:
https://github.com/endojs/endo/pull/1929/commits/19dbdd8ee06ce82d674ac0d961f920883292c4d0